webapp/channels/src/components/logged_in/logged_in.tsx

```diff
@@ -206,7 +206,11 @@ export default class LoggedIn extends React.PureComponent<Props> {
     private handleBackSpace = (e: KeyboardEvent): void => {
         const excludedElements = ['input', 'textarea'];
 
-        if (e.which === BACKSPACE_CHAR && !(excludedElements.includes((e.target as HTMLElement).tagName.toLowerCase()))) {
+        if (
+            e.which === BACKSPACE_CHAR && e.target &&
+            !(excludedElements.includes((e.target as HTMLElement).tagName.toLowerCase())) &&
+            (e.target as HTMLElement).getAttribute?.('contenteditable') !== 'true'
+        ) {
```

Comment:
would you be open in improving this code, this piece was written many years ago. We would want to do the following here to improve the readability 
- remove `which` its deprecated and replace it with `isKeyPressed(e, Constants.KeyCodes.BACKSPACE)` these are utility used in codebase
- create constants for `targetElement` and `targetsTagName`
- create `isTargetNotContentEditable` constant